src/main/java/duke/Task.java

```diff
@@ -0,0 +1,38 @@
+package duke;
+
+/**
+ * Represents a Task object.
+ */
+public abstract class Task {
+    protected final String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false; // tasks always start as not done
+    }
+
+    public boolean isDone() { return isDone; }
```

Comment:
I think the method body can put in a separate line so that it is more consistent with other method style.